app/design/frontend/base/default/template/securesubmit/storedcards.phtml

```diff
@@ -9,8 +9,9 @@ $_storedCards = Mage::helper('hps_securesubmit')->getStoredCards(Mage::getSingle
             <h2><?php echo $this->__('Saved Credit Cards') ?></h2>
 
         <ul id="hps-saved-cards" class="saved-card-table">
-            <?php foreach ($_storedCards->getData() as $card): ?>
+            
                 <li>
+                  <?php foreach ($_storedCards->getData() as $card): ?>
```

Comment:
@makenziemoore Moving this line here would create broken HTML (one opening `li`, bunch of closing `li`).

Reply:
not a show stopper, so pushing this out to next deployment